src/elements/diagnostics/BeamMonitor.cpp

```diff
@@ -126,11 +126,11 @@ namespace detail {
     {
 #ifdef ImpactX_USE_OPENPMD
         // pick first available backend if default is chosen
-        if( m_OpenPMDFileType == "default" )
+        if (m_OpenPMDFileType == "default")
 #   if openPMD_HAVE_ADIOS2==1
-        m_OpenPMDFileType = "bp";
+        m_OpenPMDFileType = "bp4";
```

Comment:
We want to change this to bp5 as soon as variable based encoding works well for our (reader) tools.